vnext/Microsoft.ReactNative/IReactDispatcher.h

```diff
@@ -38,7 +38,6 @@ struct ReactDispatcher : implements<ReactDispatcher, IReactDispatcher, Mso::Reac
   static void SetUIThreadDispatcher(IReactPropertyBag const &properties) noexcept;
 
   static IReactPropertyName JSDispatcherProperty() noexcept;
-  static IReactDispatcher GetJSDispatcher(IReactPropertyBag const &properties) noexcept;
```

Comment:
Please keep the IReactDispatcher.h and IReactDispatcher.cpp files unchanged.
Let's keep the access to the JS dispatcher as we do it for the UI Dispatcher.